metadata-models/src/main/pegasus/com/linkedin/metadata/run/RelatedAspect.pdl

```diff
@@ -0,0 +1,24 @@
+namespace com.linkedin.metadata.run
+
+import com.linkedin.common.Urn
+
+/**
+ * Metadata about how an aspect is related to some urn
+ */
+record RelatedAspect {
+
+  /**
+   *  Urn of the entity containing a related aspect
+   */
+  entity: Urn
```

Comment:
Is the entity that is associated with the aspect? Or the entity that is _referenced by_ the aspect? Maybe useful to call out in the doc.

Reply:
The entity that is referenced by the aspect.